CONTRIBUTING.md

```diff
@@ -66,6 +69,51 @@ For this method, your name and e-mail must be the same registered on your GitHub
 ### **Code Review**
 - All your submissions needs a review.
 
+### **Issues**
+
+If you have a bug or an idea, check out the following sections before submitting your contribution.
+
+
+#### **Check the issue tracker**
+
+All our issues are centralized in our main repository, it is quite likely that you will find a topic that is being discussed. Check the [**open issues**](https://github.com/ZupIT/horusec/issues), another good way to start is [**good first issues**](https://github.com/ZupIT/horusec/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22).
+
+Use the GitHub's search filter to help you. For example:
+
+- **Test related issues:** `is:open is:issue label:kind/tests `
+- **Issues that need extra attention:** `is:open is:issue label:"help wanted" `
+- **Issues related to a bug:** `is:issue is:open label:kind/bug `
+
+#### **Open an issue for any new problem**
+
+Writing a good issue will help our team better analyze and manage your contributions, therefore, follow the standards and best practices below:
+
+**With the title:**
+
+**[Project][Scope]: Title Description**  
```

Comment:
I think we can change to "Project:Scope - Tittle description" so we can follow the same pattern [here](https://github.com/ZupIT/horusec/blob/bbbb2d1f40ed0145b865c04e359179de5219e0a7/CONTRIBUTING.md#pkgtype---description)